client/src/locales/en-US.json

```
@@ -249,8 +250,8 @@
         "cancelNo": "Resume"
     },
     "JoinByLinkModal": {
-        "pageTitle": "Join by link",
-        "pleaseEnterUrl": "Please enter the organization URL",
+        "pageTitle": "Join an organization",
+        "pleaseEnterUrl": "If you have received an invitation email, copy the invitation link and paste it below.",
```

The link isn't really accessible from the email (it's an `https://` that redirects to a `parsec3://` which starts the process). A standard use case for this modal would be if the inviter manually sends the invitation link to the user.

---

The [docs](https://docs.parsec.cloud/en/stable/userguide/new_user.html#receive-invite) confused me: 
```
If clicking on the URL doesn’t work, 
you can also use the Enter a URL manually dialog.
```
So, if it's not the `https` link that should be pasted here, what do you think of: 
```suggestion
        "pleaseEnterUrl": "If you have received an invitation link, please paste it below.",
```

---

https://github.com/Scille/parsec-cloud/pull/8202/files#diff-816bb4380aa93aeca09643d55a0bdbf2267a990978f11caede416d210881740aR254